src/python/pants/backend/python/dependency_inference/subsystem.py

```diff
@@ -191,6 +191,31 @@ class PythonInferSubsystem(Subsystem):
         ),
     )
 
+    ignore_self_imports = BoolOption(
+        default=False,
+        help=softwrap(
+            """Whether self imports of a module should be ignored.
+
+            This is different from the `dependencies-closed` option which includes the input targets in the output.
```

Comment:
I suggest having this last in the help string. To me, it's not very helpful to describe a feature in terms of what it is not, but still worth keeping to highlight/motivate it's existence when there are other similar options available.

Reply:
That's a great point, thanks, Andreas. This has now been done.